demos/data/create-sqlite-db.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\login\demo;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+$srcFile = __DIR__ . '/dump.sql';
+$destFile = __DIR__ . '/db.sqlite';
+
+if (file_exists($destFile)) {
+    unlink($destFile);
+}
+
+$sqliteFile = __DIR__ . '/db.sqlite';
+$db = \atk4\data\Persistence::connect('sqlite:' . $sqliteFile);
+
+foreach (preg_split('~;\s*(\n\s*|$)~', file_get_contents($srcFile)) as $query) {
```

Comment:
drop everything with native SQL and refactor it to newer approach, see ui

Reply:
If I'm not mistaken, then this is not used anyway. I use sqlite db generated with migrator.

Reply:
In my company, I say to everyone that duplicating code = 5x more work in the future. When you not get it quickly, you are out.